dist/linux/README.md

```diff
@@ -0,0 +1,24 @@
+# Installing on Linux
+
+## Building
+```sh
+cd byedpi/
+make
+sudo make install
+```
+
+## Systemd Service (optional)
+
+Copy and enable the service:
+
+```sh
+cp byedpi.service /etc/systemd/system/byedpi.service
```

Comment:
Systemd supports user services in `~/.config/systemd/user/`. 
byedpi does not require root privileges to run. So i think user service will be much safer.

Reply:
good point. thanks!